app/controllers/resources_controller.rb

```diff
@@ -7,6 +7,11 @@ def index
                    Resource.all
                  end
 
-    render json: @resources
+    render json: @resources.to_json(include: ['addresses', 'phones', { schedule: { include: 'schedule_days' } }])
+  end
+
+  def show
+    @resources = Resource.find(params[:id])
```

Comment:
I think this can be `resource` (singular, no "@"). The `@resources` variable in `index` should probably be `resources`, too. The variables aren't referenced outside of the methods in which they're defined, so they don't need to be instance variables.